### DISCUSSION TITLE
Question about licences

### DISCUSSION THREAD

**Discussion by huapii-ben-desmedt** [JetClient/jet-client-support#68] [Q&A]
Hi,

My initial impression is really good, I like the idea to have it as an IDE plugin and to be able to add requests to the repo, it will make onboarding new devs easier. However, I have few questions.

1. What is the difference between an individual licence and one for an organization? Beyond the price, I don't find any info on the difference. Are there additional costs for more team members? For now, I'm considering get an individual licence, since our need isn't pressing, and this would give me time to fully explore the plugin, but what would be the limitations of that licence and the steps to take if we decide to use it on team level?
2. What about consultants? We sometimes have developers working temporarily on the project, can they use the plugin as intended whilst working for us, without the need to demand they get a licence ? 
3. What about sensitive data stored in variables, like passwords, client secrets for external api calls? Even though our repo is private, I'd rather not commit those, but still might want to use them in an environment to use in requests.

**Comment by AntonShuvaev** [comment]
Hi,

1. The difference between a Personal and an Organization License is similar to what you might see with JetBrains products. You can find a detailed comparison here: [JetBrains License Comparison](https://www.jetbrains.com/store/comparison.html#LicenseComparison).
- Personal License: This is for individual use and cannot be bought by companies. However, individuals can buy a license with their own money and use it at work. 
- Organization License: This license is for company use and allows multiple employees to use it, but not at the same time.

2. For consultants or temporary developers, the Organization License allows them to use the plugin while they are working on your projects without needing their own licenses. However, they cannot use it simultaneously.

3. JetClient uses "local" and "shared" variables. Local variables, intended for sensitive or temporary data, are not saved to sync files. Shared variables, on the other hand, are saved in sync files and are meant for non-sensitive data that you want to share with your team.
